keel-web/src/test/resources/application.properties

```diff
@@ -14,3 +14,5 @@ sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:somepo
 sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
 spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
 spring.application.name=keel
+keel.plugins.ec2.enabled=true
+keel.plugins.titus.enabled=true
```

Comment:
Aren't these going to apply to, and potentially affect, other tests? You can specify specific properties for the test with the `@SpringBootTest` annotation.

Reply:
It looks like all of the tests are passing. My instinct is that it's better to leave these enabled for all tests, and so I'm going to remove the cases where we explicitly enable these in tests and leave them set here.